Anytype/Sources/PresentationLayer/TextEditor/EditorPage/Models/BlockViewModelBuilder.swift

```diff
@@ -87,7 +87,7 @@ final class BlockViewModelBuilder {
             items.insert(.system(spacer), at: featureRelationsIndex + 2)
         } else if let featureRelationsIndex  {
             items.insert(.system(spacer), at: featureRelationsIndex + 1)
-        } else if let openFileButton  {
+        } else if let openFileButton, items.count > 0 {
```

Comment:
interesting, how it can be `0` if we have this line above `var items = viewModels.map(EditorItem.block)`?

Reply:
Sometimes block subscription return empty list on updates